src/components/ThreeCanvas.js

```diff
@@ -97,7 +60,7 @@ export const ThreeCanvas = ({ imageFile, locData, driverColor, driverSelected, f
     lo.load('/map/bahrain.gltf', gltf => {
       map = gltf.scene;
       map.scale.set(0.1, 0.1, 0.1);
-      map.rotation.x = Math.PI / 2;
+      map.rotation.x = Math.PI / 2; // causing orbit issue in canvas
```

Comment:
if you comment this line out the map lays flat as it should by default, and orbit controls are now how I imagine they should be. 

when uncommented and the track is rotated up, the orbitcontrols seem unnatural. i know you have a lot of work tied into the car motion already. is it easy to get the car to move along the "natural floor" instead of up in the air where we then have to rotate the map up? let me know if you want meet on this one.